### PR TITLE
created model, migration and seeders for reviews table

### PR DESCRIPTION
**Pull request by 360devin360**
created model, migration and seeder files for reviews table as well as updated some validations for the spots tables.

### CODE REVIEW

File: backend/db/seeders/20240521075706-demo-Review.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    /**
+     * Add seed commands here.
+     *
+     * Example:
+     * await queryInterface.bulkInsert('People', [{
+     *   name: 'John Doe',
+     *   isBetaMember: false
+     * }], {});
+    */
+   await Review.bulkCreate([
```
Comment (AnthonyBronca) [inline backend/db/seeders/20240521075706-demo-Review.js:21]: you might need to add options

File: backend/db/seeders/20240521075706-demo-Review.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
+const {Review} = require('../models');
+//define schema
+let options = {}
```
Comment (AnthonyBronca) [inline backend/db/seeders/20240521075706-demo-Review.js:5]: you might need to add:

```js
options.tableName = "Reviews";
```

**Review by AnthonyBronca — COMMENTED** [review]
Looks good! A  few areas that may cause issues in production

File: backend/db/seeders/20240521075706-demo-Review.js (hunk 2 above)
Comment (360devin360) [reply backend/db/seeders/20240521075706-demo-Review.js:5]: Screenshot 2024-05-23 004837

**Pull request merged**